packages/manager/modules/vps/src/migration/vps-migration.less

```diff
@@ -0,0 +1,11 @@
+.vps-migration {
```

Comment:
Scope to the `ovh-manager-vps-migration` so that we do not face any issues even if `vps-migration` class exists elsewhere